packages/compass-import-export/src/modules/import.ts

```diff
@@ -252,6 +282,7 @@ export const startImport = () => {
     const fileSize = fileStats?.size || 0;
 
     log.info(
+      // eslint-disable-next-line @mongodb-js/compass/unique-mongodb-log-id
```

Comment:
I'm keeping the same log ids, at least for now while the logs are unchanged. So I had to add some linting and other comments to make the checks happy.